maven/plan.sh

```diff
@@ -1,19 +1,19 @@
 pkg_origin=core
 pkg_name=maven
-pkg_version=3.3.9
+pkg_version=3.6.1
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_description="A software project management and comprehension tool"
 pkg_upstream_url="https://maven.apache.org/"
 pkg_license=("Apache-2.0")
 pkg_source=http://apache.cs.utah.edu/maven/maven-3/${pkg_version}/source/apache-maven-${pkg_version}-src.tar.gz
-pkg_shasum=9150475f509b23518e67a220a9d3a821648ab27550f4ece4d07b92b1fc5611bc
+pkg_shasum=025921fff6ba827a25413ffc08fb1933565eb1f07ee2d3f228911913ee4f3c3f
 pkg_dirname="apache-$pkg_name-$pkg_version"
 pkg_deps=(
   core/coreutils
   core/jdk8
   core/which
 )
-pkg_build_deps=(core/ant)
+pkg_build_deps=(core/maven/3.3.9)
```

Comment:
There should't be any reason to pin here. As this gets updated we want the N-1 build of Maven to do the builds.

Reply:
Fixed! No more pin.